pages/services/couchbase/1.0.0-6.0.0/release-notes/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes
+title: Release Notes
+menuWeight: 130
+excerpt: Discover the new features, updates, and known limitations in this release of the Couchbase Service
+featureMaturity:
+enterprise: false
+model: /services/couchbase/data.yml
+render: mustache
+---
+
+# Release Notes for {{ model.techName }} Service version 1.0.0-6.0.0
+
+## Updates
+* Upgrade the base tech version from 5.5.3 to 6.0.0.
+
+## New Features
+* Runs on strict cluster with nobody user.
```

Comment:
Rewrite final sentence. "Runs on cluster with strict permissions and user `nobody`."

Reply:
Done.